scenes/game/game.gd

```diff
@@ -1150,6 +1150,15 @@ func _stat_notice_event(event):
 	spawn_damage_popup(notice_text, player)
 	return SmallNoticeDelay
 
+func _set_card_bonus(card_id, bonus, value=true):
+	var card = find_card_on_board(card_id)
+	if bonus == "ex":
+		card.set_ex(value)
+	if bonus == "wild":
```

Comment:
kind of weird to have 2 ifs and one elif.
I mean, it won't matter, but just odd.